thebeat/stats.py

```diff
@@ -637,17 +650,23 @@ def fft_values(
 
     # Make a sequence of ones and zeroes
     timeseries = thebeat.helpers.sequence_to_binary(sequence, resolution=step_size)
-    duration = sequence.duration
+    timeseries = timeseries if sequence.end_with_interval is True else timeseries[:-1]
```

Comment:
Is `helpers` public, or just internal?
If it's public and documented, maybe this "include final event" could be an optional keyword argument to `sequence_to_binary`? I can imagine this to be nice bonus if people want to use this?

Reply:
The way it was changed wrt. the time of the first onset, this isn't relevant anymore.